sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.0-beta.2 (2023-03-11)
 
 ### Other Changes
 
+- Upgraded dependent `Azure.Core` to `1.30.0`.
```

Comment:
@AlexanderSher what does this mean to folks who read this changelog?

Reply:
To me it would mean that something has changed in `Azure.Core` so that it is important to update the reference. Then I will look into `Azure.Core` changelog to find out what has changed.